### ISSUE TITLE
Versions above 4.19 9599 Again have the bug to consume 100% cpu.

### ISSUE THREAD

**Issue by 0bo** [SoftEtherVPN/SoftEtherVPN#234]
Today I tried the newerest version v4.21-9613-beta, after l2tp connection from win10, cpu usage rise to 99% and never drop down unless stop service.

this kind of issue happened before when using SecureNAT, but I believe it had been fixed on SoftEther VPN 4.19 Build 9582 Beta.

I also tried v4.20-rtm and v 4.19 Build 9605 Beta, they all have the same problem.

when I came back to  4.19 Build 9599 Beta (October 19, 2015), everything is working fine.

I have tested these on both ubuntu 16.04 and 14.04. Don't think it relate to the Ubuntu version.

Any idea? thanks!

**Comment by chipitsine** [comment]
can you please perform "gprof" report ? "strace" report ?
and describe steps to reproduce

**Comment by colingpt** [comment]
@chipitsine 
hi there, sorry for my limited knowledge, I just checked what do "gprof" and "strace" mean, and I will try it later.

to reproduce, I did try it on a vps with Ubuntu 16.04 from vpsbit.com, based on a tutorial from [lincoln](http://blog.lincoln.hk/blog/2013/03/19/softether-on-vps/)  , after all installation, I tried to connect it by L2TP, if there was 100% cpu situation, I then downloaded another version to try until I  got all above results.

I will try it on more vps providers later to verify if everything is ok.

thanks
